components/director/internal/domain/document/service.go

```diff
@@ -46,10 +48,15 @@ func (s *service) List(ctx context.Context, applicationID string, pageSize *int,
 }
 
 func (s *service) Create(ctx context.Context, applicationID string, in model.DocumentInput) (string, error) {
+	tnt, err := tenant.LoadFromContext(ctx)
+	if err != nil {
```

Comment:
error is descriptive enough already, wrapping does not introduce any new information